### ISSUE TITLE
Statistics/Usage should not sent out by default

### ISSUE THREAD

**Issue by gitd8b** [marius-wieschollek/passwords#537]
**Current Status**
The passwords app should only send out statistics to passwordsapp.org when the "Server survey participation" is set to basic or full (i.e. opt-in)

**Feature Description**
I noticed my server made a DNS request to statistics.passwordsapp.org today. I don't know if this is due to the fact I enabled nightly builds as part of the fix for Nextcloud 25, or if this is a new thing, but it's really discouraging to see it happening without permission of the users. Opt-in would be preferred among myself and I imagine others. 

**Additional context**
DNS Logs caught:
```
dnsmasq[30576]: 258083 ServerIP/ServerPort query[SOA] passwordsapp.org from ServerIP
dnsmasq[30576]: 258083 ServerIP/ServerPort forwarded passwordsapp.org to DNSServerIP
dnsmasq[30576]: 258085 ServerIP/ServerPort query[A] statistics.passwordsapp.org from ServerIP
dnsmasq[30576]: 258085 ServerIP/ServerPort forwarded statistics.passwordsapp.org to DNSServerIP
dnsmasq[30576]: 258085 ServerIP/ServerPort reply statistics.passwordsapp.org is 91.53.103.134
dnsmasq[30576]: 258086 ServerIP/ServerPort query[AAAA] statistics.passwordsapp.org from ServerIP
dnsmasq[30576]: 258086 ServerIP/ServerPort forwarded statistics.passwordsapp.org to DNSServerIP
dnsmasq[30576]: 258086 ServerIP/ServerPort reply statistics.passwordsapp.org is NODATA-IPv6
dnsmasq[30576]: 258087 ServerIP/ServerPort query[CNAME] statistics.passwordsapp.org from ServerIP
dnsmasq[30576]: 258087 ServerIP/ServerPort forwarded statistics.passwordsapp.org to DNSServerIP
dnsmasq[30576]: 258948 PersonalDeviceIP/PersonalDevicePort query[A] passwordsapp.org from PersonalDeviceIP
dnsmasq[30576]: 258948 PersonalDeviceIP/PersonalDevicePort forwarded passwordsapp.org to DNSServerIP
dnsmasq[30576]: 258948 PersonalDeviceIP/PersonalDevicePort reply passwordsapp.org is 91.53.103.134
```

**Comment by marius-wieschollek** [comment]
Statistics are Opt-In. When the app is first installed, it should create a notification for admins where they can opt-in to sending the statistics.

However it seems that you were right with your suspicion and the process is broken in NC25 and basic data is sent for everyone who did not make any selection but uses the app. It seems that there were also additional issues which may be the reason that the data is sent daily instead of weekly. (This can currently also be confirmed by viewing the statistics)

I have pushed an update which should restore the original behavior.

I will also delete any data collected from the affected 2022.11 nightly versions from the statistics server and block the affected versions from having data collected

**Comment by gitd8b** [comment]
Ah, awesome - that'd at least explain why I didn't see any alerts before hand, haha. I really appreciate the transparency and taking the time to look into that after my report - thank you!

**Comment by marius-wieschollek** [comment]
This should now be fixed with 2022.11.0
